rosdep/python.yaml

```diff
@@ -2632,6 +2632,19 @@ python-mechanize:
   gentoo: [dev-python/mechanize]
   nixos: [pythonPackages.mechanize]
   ubuntu: [python-mechanize]
+python-mediapipe-pip:
```

Comment:
did you wanted to add the python2 version or actually one for python3?
according to the [contributing guidelines](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#python-rules), the prefix should be `python3` and not `python` in that case.

AFAIS `mediapipe` never supported python2

Reply:
Thy for the hint. I updated the prefix accordingly.